hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/validate/HazelcastSqlOperatorTable.java

```diff
@@ -263,6 +267,71 @@
 
     //#endregion
 
+    //#region Math functions.
+
+    public static final SqlFunction ABS = new SqlFunction(
+        "ABS",
+        SqlKind.OTHER_FUNCTION,
+        HazelcastReturnTypes.UNARY_MINUS,
+        HazelcastInferTypes.DECIMAL_IF_UNKNOWN,
+        notAny(OperandTypes.NUMERIC_OR_INTERVAL),
+        SqlFunctionCategory.NUMERIC
+    );
+
+    public static final SqlFunction SIGN = new SqlFunction(
+        "SIGN",
+        SqlKind.OTHER_FUNCTION,
+        ReturnTypes.ARG0,
+        HazelcastInferTypes.DECIMAL_IF_UNKNOWN,
+        notAny(OperandTypes.NUMERIC),
+        SqlFunctionCategory.NUMERIC
+    );
+
+    public static final SqlFunction RAND = new SqlFunction(
+        "RAND",
+        SqlKind.OTHER_FUNCTION,
+        ReturnTypes.DOUBLE,
+        HazelcastInferTypes.DECIMAL_IF_UNKNOWN,
+        OperandTypes.or(OperandTypes.NILADIC, notAny(OperandTypes.NUMERIC)),
+        SqlFunctionCategory.NUMERIC
+    );
+
+    public static final SqlFunction COS = new HazelcastDoubleFunction("COS");
+    public static final SqlFunction SIN = new HazelcastDoubleFunction("SIN");
+    public static final SqlFunction TAN = new HazelcastDoubleFunction("TAN");
+    public static final SqlFunction COT = new HazelcastDoubleFunction("COT");
+    public static final SqlFunction ACOS = new HazelcastDoubleFunction("ACOS");
+    public static final SqlFunction ASIN = new HazelcastDoubleFunction("ASIN");
+    public static final SqlFunction ATAN = new HazelcastDoubleFunction("ATAN");
+    public static final SqlFunction EXP = new HazelcastDoubleFunction("EXP");
+    public static final SqlFunction LN = new HazelcastDoubleFunction("LN");
+    public static final SqlFunction LOG10 = new HazelcastDoubleFunction("LOG10");
+    public static final SqlFunction DEGREES = new HazelcastDoubleFunction("DEGREES");
+    public static final SqlFunction RADIANS = new HazelcastDoubleFunction("RADIANS");
+
+    public static final SqlFunction FLOOR = new HazelcastSqlFloorFunction(SqlKind.FLOOR);
+    public static final SqlFunction CEIL = new HazelcastSqlFloorFunction(SqlKind.CEIL);
+
+    public static final SqlFunction ROUND = new SqlFunction(
+        "ROUND",
+        SqlKind.OTHER_FUNCTION,
+        ReturnTypes.ARG0_NULLABLE,
+        HazelcastInferTypes.ROUND,
+        OperandTypes.NUMERIC_OPTIONAL_INTEGER,
+        SqlFunctionCategory.NUMERIC
+    );
+
+    public static final SqlFunction TRUNCATE = new SqlFunction(
+        "TRUNCATE",
+        SqlKind.OTHER_FUNCTION,
+        ReturnTypes.ARG0_NULLABLE,
+        HazelcastInferTypes.ROUND,
+        OperandTypes.NUMERIC_OPTIONAL_INTEGER,
```

Comment:
notAny?

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/17340/commits/95442d9b91abe7324b404b3fee044505319410ce